packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -412,8 +412,15 @@ class RenderParagraph extends RenderBox
         transform: transform,
         position: position,
         hitTest: (BoxHitTestResult result, Offset transformed) {
-          transform.invert();
-          assert(transformed == MatrixUtils.transformPoint(transform, position));
+          assert(() {
+            final Offset manualPosition = (position - textParentData.offset) / textParentData.scale;
+            // Compare the two offsets ignoring floating point error.
+            final double epsilon = 0.00001;
```

Comment:
Use https://master-api.flutter.dev/flutter/foundation/precisionErrorTolerance-constant.html instead of defining your own.

Reply:
Ahh thanks! was looking for something like this!